src/graphqlProtocol/resolver.ts

```diff
@@ -453,3 +457,170 @@ export const getMutationResolver = (
     }
   };
 };
+
+export const getSubscriptionResolver = (
```

Comment:
Function used to generate the AsyncIterator that will be used to retrieve messages published by pubsub provided by Shyft or context.